hbase-client/src/main/java/org/apache/hadoop/hbase/client/Mutation.java

```diff
@@ -341,17 +328,6 @@ public int compareTo(final Row d) {
     return Bytes.compareTo(this.getRow(), d.getRow());
   }
 
-  /**
-   * Method for retrieving the timestamp
-   * @return timestamp
-   * @deprecated As of release 2.0.0, this will be removed in HBase 3.0.0.
-   *             Use {@link #getTimestamp()} instead
```

Comment:
This is odd. It seems to be telling me use this method -- i.e. the one that is deprecated -- instead.... but I don't think it impinges upon this effort....

Reply:
Something about getTimeStamp vs getTimestamp?

Reply:
Yes. We had this discussion a while ago and changed it to `xxxTimestamp`.